dotCMS/src/integration-test/java/com/dotcms/util/MimeTypeUtilsTest.java

```diff
@@ -0,0 +1,86 @@
+package com.dotcms.util;
+
+import com.dotcms.UnitTestBase;
+import com.dotcms.contenttype.business.ContentTypeAPIImpl;
+import com.dotcms.datagen.TestDataUtils;
+import com.dotcms.datagen.TestUserUtils;
+import com.dotcms.repackage.org.apache.commons.io.FileUtils;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.business.RoleAPI;
+import com.dotmarketing.portlets.workflows.actionlet.PublishContentActionlet;
+import com.dotmarketing.portlets.workflows.actionlet.SaveContentActionlet;
+import com.dotmarketing.portlets.workflows.actionlet.SaveContentAsDraftActionlet;
+import com.dotmarketing.portlets.workflows.actionlet.UnpublishContentActionlet;
+import com.dotmarketing.portlets.workflows.business.BaseWorkflowIntegrationTest;
+import com.dotmarketing.portlets.workflows.model.WorkflowState;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.util.StringPool;
+import com.twelvemonkeys.imageio.plugins.jpeg.JPEGImageWriter;
+import io.vavr.Tuple;
+import io.vavr.Tuple2;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static com.dotcms.util.CollectionsUtils.*;
+import static org.junit.Assert.*;
+
+/**
+ * MimeTypeUtils unit test.
+ * @author jsanca
+ */
+public class MimeTypeUtilsTest extends UnitTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4772115)